extensions/INTEL/SPV_INTEL_fpga_buffer_location.asciidoc

```diff
@@ -0,0 +1,122 @@
+SPV_INTEL_fpga_buffer_location
+==============================
+
+== Name Strings
+
+SPV_INTEL_fpga_buffer_location
+
+== Contact
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Headers
+
+== Contributors
+
+- Joe Garvey, Intel +
+
+== Notice
+
+Copyright (c) 2023 Intel Corporation.  All rights reserved.
+
+== Status
+
+Final Draft
+
+== Version
+
+[width="40%",cols="25,25"]
+|========================================
+| Last Modified Date | 2023-02-01
+| Revision           | 1
+|========================================
+
+== Dependencies
+
+This extension is written against the SPIR-V Specification,
+Version 1.6 Revision 2.
+
+This extension requires SPIR-V 1.0.
+
+== Overview
+
+This extension adds a function parameter decoration that is useful for FPGA targets.  This decoration indicates that a particular global memory pointer can only access a particular physical memory location.  Knowing this information at compile time can allow FPGA compilers to generate load store units of lower area for accesses done through such a pointer.  
```

Comment:
Taking an account https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1801/files - the overview should be updated

Reply:
Right. I think this should be both function parameter decoration and Instruction parameter decoration.

There are some history here. The function parameter decoration is already checked in into the code but the spec is not yet posted at that time... So this spec actually covers both

Reply:
Please review the recent commit to see if the displayed is appropriate. Thanks!

Reply:
Moved the discussion internally. Once internal discussion resolved please don't forget to update generated .html file here.

Reply:
Adjusted the docs in this PR after internal adjustments.